src/main/java/com/eerussianguy/firmalife/CommonEventHandlerFL.java

```diff
@@ -50,6 +57,12 @@ public static void onBlockHarvestDrops(BlockEvent.HarvestDropsEvent event)
         {
             event.getDrops().add(new ItemStack(ItemsFL.FRUIT_LEAF, 2 + Constants.RNG.nextInt(4)));
         }
+        else if (block instanceof BlockFruitTreeTrunk)
+        {
+            IFruitTree tree = ((BlockFruitTreeTrunk) block).getTree();
+            String poleName = MOD_ID + tree.getName().toLowerCase() + "_pole";
+            event.getDrops().add(new ItemStack(ItemMisc.getByNameOrId(poleName)));
```

Comment:
This is bad practice but I'm going to let it slide for now. Might be possible to do BlockFruitTreeTrunk.get()? Check that